test/README.md

```diff
@@ -0,0 +1,15 @@
+# Running tests
+
+This folder has all the system test files for lxd.
+
+## Dependencies
+
+You'll need the following dependencies installed before you can run them:
+
+    sudo apt-get install jq sqlite3 golang-go
+
+
+## Running
+
+A simple "cd test && sudo ./main.sh" should be all you need to run the system
```

Comment:
should be "sudo -E ./main.sh" otherwise you'll be testing your system lxc/lxd instead of the one from your GOPATH.